maps/kondaru.dmm

```diff
@@ -27931,7 +27816,7 @@
 "bCR" = (
 /obj/table/wood/auto,
 /obj/displaycase{
-	displayed = new /obj/item/captaingun
```

Comment:
Looks like the old iteration was broken because it *didn't* include a semi-colon despite having 2+ properties. I'm not super-familiar with .dmm format so unsure *if* this is what's causing the issue, but given the error type and location, and that it generated the after.png (which *has* the `;`), I guess?

Reply:
Found where this change was made. PR #4000 altered maps due to a display case change, and the new displayed object var was edited into place without the semicolon afterwards.

Reply:
Cool. Feel free to PR a fix for that and then we can try to see how this looks with before/after.

Reply:
Correct, it needs the semi colon or to have the pixel offset removed

> (which would absolutely not be part of this PR).
also correct, it would to be merged prior. I'll go ahead and do that for you.

Reply:
PR'd as requested. #4283